### PR TITLE
Add MSP2_UBLOX_REQUEST_SV_INFO

### PR DESCRIPTION
**Pull request by haslinghuis**
See https://github.com/betaflight/betaflight/pull/13240

Should fix configurator missing satellite data request more reliable.

- Adds `MSP2_UBLOX_REQUEST_SV_INFO` request to activate SatInfo polling in firmware.
- SatInfo will be turned off upon ARMING the quad.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Pull request merged**
